packages/html-reporter/src/metadataView.tsx

```diff
@@ -18,21 +18,62 @@ import * as React from 'react';
 import './colors.css';
 import './common.css';
 import * as icons from './icons';
-import type { Metadata } from './index';
 import { AutoChip } from './chip';
 import './reportView.css';
 import './theme.css';
 
-export const MetadataView: React.FC<Metadata> = metadata => {
+interface Info {
+  'revision.id'?: string;
+  'revision.author'?: string;
+  'revision.email'?: string;
+  'revision.subject'?: string;
+  'revision.timestamp'?: number | Date;
+  'revision.link'?: string;
+  'ci.link'?: string;
+}
+
+export type GitCommitInfo =  (Info & { timestamp?: number }) | undefined;
+
+export class ErrorBoundary extends React.Component<{}, { error: Error | null, errorInfo: React.ErrorInfo | null }> {
```

Comment:
I would not want to encourage this kind of try/catch behavior, what's wrong with a local try/catch where you are validating user-provided data?